clients/src/test/java/org/apache/kafka/clients/producer/KafkaProducerTest.java

```diff
@@ -850,7 +851,7 @@ public void testMetadataTimeoutWithMissingTopic(boolean isIdempotenceEnabled) th
         try {
             future.get();
```

Comment:
Does it expect to get `ExecutionException`? if so, could you please rewrite it by `assertThrow`?

Reply:
yep, we are on the same page :)